frontend/src/metabase/visualizations/visualizations/PieChart/utils.ts

```diff
@@ -5,13 +5,9 @@ export function getMaxLabelDimension(
   d3Arc: d3.svg.Arc<d3.svg.arc.Arc>,
   slice: d3.svg.arc.Arc,
 ) {
-  // Invalid typing
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
+  // @ts-expect-error Invalid typing
   const innerRadius = d3Arc.innerRadius()();
-  // Invalid typing
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
+  // @ts-expect-error Invalid typing
```

Comment:
Incorrect library typings are not expected. Suppressing TS here is a workaround.